addons/l10n_fr_fec/wizard/account_fr_fec.py

```diff
@@ -25,62 +25,66 @@ class AccountFrFec(models.TransientModel):
     export_type = fields.Selection([
         ('official', 'Official FEC report (posted entries only)'),
         ('nonofficial', 'Non-official FEC report (posted and unposted entries)'),
-        ], string='Export Type', required=True, default='official')
+    ], string='Export Type', required=True, default='official')
+    excluded_journal_ids = fields.Many2many('account.journal', string="Excluded Journals", domain="[('company_id', '=', current_company_id)]")
 
     @api.onchange('test_file')
     def _onchange_export_file(self):
         if not self.test_file:
             self.export_type = 'official'
 
+    def _get_where_query(self):
+        where_params = {'company_id': self.env.company.id}
+        where_query = "am.company_id = %(company_id)s\n"
+        # For official report: only use posted entries
+        if self.export_type == "official":
```

Comment:
Why do you prefer checking the export type instead of test_file? Simple question not asking for change